endpoints/getting-started-grpc/README.md

```diff
@@ -95,7 +95,6 @@ Cloud account and [SDK](https://cloud.google.com/sdk/) configured.
     ```bash
```

Comment:
Up above after "gcloud endpoints services deploy ...":
- Remove comment about noting the config ID.
- Remove command about getting the config ID.